*** apm_edit.ui.orig	2009-08-09 08:37:24.000000000 -0400
--- apm_edit.ui	2009-08-09 08:37:42.000000000 -0400
***************
*** 467,478 ****
          <action name="moduleEditAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <actiongroup name="InsertActionGroup"/>
!         <item text="&amp;Insert" name="popupMenu_33" accel="">
!             <action name="moduleInsertModuleAction"/>
!             <action name="moduleInsertSubmodelAction"/>
!             <action name="moduleInsertAsRootAction"/>
!         </item>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Alternatives" name="alternativesMenu">
--- 467,473 ----
          <action name="moduleEditAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <action name="InsertActionGroup"/>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Alternatives" name="alternativesMenu">
