*** apm_edit.ui~	2010-12-14 16:56:30.000000000 -0500
--- apm_edit.ui	2010-12-14 16:58:54.320355000 -0500
***************
*** 493,504 ****
          <action name="moduleShell_at_containing_folderAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <actiongroup name="InsertActionGroup"/>
!         <item text="&amp;Insert" name="popupMenu_38" accel="">
!             <action name="moduleInsertModuleAction"/>
!             <action name="moduleInsertSubmodelAction"/>
!             <action name="moduleInsertAsRootAction"/>
!         </item>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Alternatives" name="alternativesMenu">
--- 493,499 ----
          <action name="moduleShell_at_containing_folderAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <action name="InsertActionGroup"/>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Alternatives" name="alternativesMenu">
