*** apm_edit.ui.orig	2009-08-04 12:07:44.000000000 -0400
--- apm_edit.ui	2009-08-04 12:09:43.000000000 -0400
***************
*** 422,433 ****
          <action name="moduleEditAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <actiongroup name="InsertActionGroup"/>
!         <item text="&amp;Insert" name="popupMenu_33" accel="">
!             <action name="moduleInsertModuleAction"/>
!             <action name="moduleInsertSubmodelAction"/>
!             <action name="moduleInsertAsRootAction"/>
!         </item>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Help" name="helpMenu">
--- 422,428 ----
          <action name="moduleEditAction"/>
          <action name="moduleNotesAction"/>
          <separator/>
!         <action name="InsertActionGroup"/>
          <action name="moduleDeleteAction"/>
      </item>
      <item text="&amp;Help" name="helpMenu">
***************
*** 493,499 ****
              <cstring>fileNewAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>New</string>
--- 488,494 ----
              <cstring>fileNewAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>filenew.png</iconset>
          </property>
          <property name="text">
              <string>New</string>
***************
*** 510,516 ****
              <cstring>fileOpenAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Open</string>
--- 505,511 ----
              <cstring>fileOpenAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>fileopen.png</iconset>
          </property>
          <property name="text">
              <string>Open</string>
***************
*** 527,533 ****
              <cstring>fileSaveAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Save</string>
--- 522,528 ----
              <cstring>fileSaveAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>filesave.png</iconset>
          </property>
          <property name="text">
              <string>Save</string>
***************
*** 558,564 ****
              <cstring>filePrintAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Print</string>
--- 553,559 ----
              <cstring>filePrintAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>print.png</iconset>
          </property>
          <property name="text">
              <string>Print</string>
***************
*** 589,595 ****
              <cstring>editUndoAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Undo</string>
--- 584,590 ----
              <cstring>editUndoAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>undo.png</iconset>
          </property>
          <property name="text">
              <string>Undo</string>
***************
*** 606,612 ****
              <cstring>editRedoAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Redo</string>
--- 601,607 ----
              <cstring>editRedoAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>redo.png</iconset>
          </property>
          <property name="text">
              <string>Redo</string>
***************
*** 623,629 ****
              <cstring>editCutAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Cut</string>
--- 618,624 ----
              <cstring>editCutAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>editcut.png</iconset>
          </property>
          <property name="text">
              <string>Cut</string>
***************
*** 640,646 ****
              <cstring>editCopyAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Copy</string>
--- 635,641 ----
              <cstring>editCopyAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>editcopy.png</iconset>
          </property>
          <property name="text">
              <string>Copy</string>
***************
*** 657,663 ****
              <cstring>editPasteAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Paste</string>
--- 652,658 ----
              <cstring>editPasteAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>editpaste.png</iconset>
          </property>
          <property name="text">
              <string>Paste</string>
***************
*** 674,680 ****
              <cstring>editFindAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>Find</string>
--- 669,675 ----
              <cstring>editFindAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>searchfind.png</iconset>
          </property>
          <property name="text">
              <string>Find</string>
***************
*** 834,840 ****
              <cstring>helpWhatsThisAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset></iconset>
          </property>
          <property name="text">
              <string>What's this</string>
--- 829,835 ----
              <cstring>helpWhatsThisAction</cstring>
          </property>
          <property name="iconSet">
!             <iconset>whatsthis.png</iconset>
          </property>
          <property name="text">
              <string>What's this</string>
***************
*** 1408,1414 ****
      <slot>editFindMissing()</slot>
      <slot>modelNuke()</slot>
      <slot>modelConfigure()</slot>
-     <slot>modelCleanAction_activated()</slot>
      <slot>modelBuild()</slot>
      <slot>modelSetupAction_activated()</slot>
      <slot>modelRunAction_activated()</slot>
--- 1403,1408 ----
***************
*** 1447,1452 ****
--- 1441,1447 ----
      <slot>ParamChange_clicked()</slot>
      <slot returnType="int">Model_find()</slot>
      <slot returnType="int *">Model_walk()</slot>
+     <slot>modelCleanAction_activated()</slot>
  </slots>
  <functions>
      <function access="private">modelOperation()</function>
